src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx

```diff
@@ -169,7 +169,7 @@ export class ConversationSnippet extends React.Component<Props, any> {
                   <MetadataText>
                     {date}
                   </MetadataText>
-                  <UnreadIndicator />
+                  {conversation.is_last_message_to_user && !conversation.last_message_open && <UnreadIndicator />}
```

Comment:
Well, that's a string/timestamp of the open, so checking the initial boolean should suffice.

Aka, it doesn't matter if `last_message_open` is `null` or not. So if the last message is to the user, and it hasn't been opened yet- show the indicator. Any other cases- we wouldn't want to show it.